src/connector/src/sink/iceberg/prometheus/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod monitor_base_file_writer;
```

Comment:
monitor -> monitored

Reply:
Besides, the struct defined in these mods can be renamed to `MonitoredXxxWriter` as well.